infra/modules/providers/azure/storage-account/variables.tf

```diff
@@ -0,0 +1,62 @@
+variable "resource_group_name" {
+  description = "The name of the resource group."
+  type        = "string"
+}
+
+variable "resource_group_location" {
+  description = "The location of the resource group."
+  type        = "string"
+}
+
+variable "account_name" {
+  description = "The name of the storage account service."
+  type        = "string"
+}
+
+variable "storage_container_name" {
+  description = "The name of the storage container. Must be unique under the storage account."
+  type        = "string"
+}
+
+variable "performance_tier" {
+  description = "Determines the level of performance required."
+  type        = "string"
```

Comment:
FYI, `type = string` is the TF 12 way to do this, though `type = "string"` also works. More syntactically TF-12 though :)

Reply:
Yeah, we'll do it the TF 12 way. Thanks!